test/helpers.test.js

```diff
@@ -181,19 +181,27 @@ describe('Speech helper methods', () => {
       // Stub the underlying _streamingRecognize method to just return
       // a bogus stream.
       var requestStream = new stream.PassThrough({objectMode: true});
+
       sandbox
         .stub(client._innerApiCalls, 'streamingRecognize')
         .returns(requestStream);
 
       var userStream = client.streamingRecognize(CONFIG, OPTIONS);
       var audioContent = Buffer.from('audio content');
 
+      var count = 0;
       requestStream._write = (data, enc, next) => {
-        assert.deepStrictEqual(data, {
-          audioContent: audioContent,
-          streamingConfig: CONFIG,
-        });
-        setImmediate(done);
+        if (count === 0)
```

Comment:
Not sure how to stub this probably - tried with `sandbox.stub(..).onFirstCall()..onSecondCall()`, the code path doesn't end up into the stub